### ISSUE TITLE
Improve the text in the codelab- Moving Augmented Images -https://codelabs.developers.google.com/codelabs/augimg-intro/index.html?index=..%2F..index#0 

### ISSUE THREAD

**Issue by hmaw** [googlecodelabs/arcore-augmentedimage-intro#2]
There are several garbled text in this codelab that needs to be adjusted for better reading clarity.
If this was created by someone that this as their second language, then I am impressed as they have done an amazing job, but it needs a little more help.  No one really really masters the English language, even those that only use it.   

(QUESTION: This might be related to another issue on the same page.
I opened this as a separate issue as the my first report was a survey question, and these are plain text changes. )

1. 
What you'll need
Make sure you have there before starting this codelab:

Change the text to:
Make sure you have everything that you need before starting this codelab:

2.
A supported ARCore device, connected via a USB cable to your development machine.
No comma needed after the word device.

3.
Access to the internet, for downloading libraries during development.
Change to:  
Access to the internet, as you will need to download libraries during development.

**Comment by kai-dream** [comment]
Good point. All updated as you suggested. Thanks!
